src/Sylius/Bundle/CoreBundle/EventListener/ChannelDeletionListener.php

```diff
@@ -37,8 +35,6 @@ public function __construct(ChannelRepository $repository)
     }
 
     /**
-     * Prevent channel deletion if no more channels enabled.
```

Comment:
I would leave these comments since the method name and class name does not explain that.